src/components/molecules/DepartmentManagement/DepartmentManagement.tsx

```diff
@@ -0,0 +1,60 @@
+import { FC, useCallback } from 'react'
+import { useTranslation } from 'react-i18next'
+import {
+  useDepartmentsFetch,
+  useParallelMutationDepartment,
+} from 'hooks/requests/useDepartments'
+
+import EditableListContainer from 'components/molecules/EditableListContainer/EditableListContainer'
+import { ModalTypes, showModal } from 'components/organisms/modals/ModalRoot'
+import { EditDataType } from 'components/organisms/modals/EditableListModal/EditableListModal'
+import { showNotification } from 'components/organisms/NotificationRoot/NotificationRoot'
+import { NotificationTypes } from '../Notification/Notification'
```

Comment:
**[non-blocker]**

Not too important, but would be good to use absolute imports everywhere (it auto-import with relative import if you are in the same subfolder under components).

Won't change much for now, but we might start reorganizing the files at some point and it's a lot easier to do with absolute imports